src/curl.cpp

```diff
@@ -346,7 +346,7 @@ long             S3fsCurl::connect_timeout     = 300;  // default
 time_t           S3fsCurl::readwrite_timeout   = 60;   // default
 int              S3fsCurl::retries             = 3;    // default
 bool             S3fsCurl::is_public_bucket    = false;
-string           S3fsCurl::default_acl         = "private";
+string           S3fsCurl::default_acl         = "";
```

Comment:
@andrewgaul 
I am planning to release v1.81 soon.
I definitely would like to incorporate this fix, but could you set default to output headers with values as private in the same way as before?
(Because this default change is scary of the impact.)